src/modules/user/user.service.ts

```diff
@@ -291,33 +292,74 @@ export class UserService {
   }
 
   async getFheelancerInformation(
-    take: number,
-    page: number,
-    search: string,
+    params: GetFreelancerDto,
   ): Promise<[User[], number]> {
     try {
-      const currentUser = await this.userRepository
+      const {
+        page,
+        take,
+        skills,
+        categories,
+        hourly_rate_start,
+        hourly_rate_end,
+        search,
+        ...userPayload
+      } = params;
+      const role = 'Freelancer';
```

Comment:
why this constant here. I think you need to have some general constant, I don't think you use `Freelancer` only here

Reply:
fixed